lang/luasocket5.3/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2009-2013 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luasocket
+PKG_SOURCE_VERSION:=733af884f1aa18ff469bf3c4d18810e815853211
+PKG_VERSION:=3.0-rc1-20190421
```

Comment:
Remove PKG_VERSION and add PKG_SOURCE_DATE. Use https://github.com/openwrt/packages/blob/master/utils/usbmuxd/Makefile as a guide

Reply:
I've switched to PKG_SOURCE_DATE